tests/Cookie/SessionCookieJarTest.php

```diff
@@ -36,9 +36,9 @@ public function testLoadsFromSession()
         unset($_SESSION[$this->sessionVar]);
     }
 
-    public function testPersistsToSession()
+    public function testPersistsToSession($testSaveSessionCookie = false)
```

Comment:
This should have a `@dataProvider` that invokes the test. See https://phpunit.de/manual/current/en/appendixes.annotations.html#appendixes.annotations.dataProvider